admin/src/pages/Creation.vue

```diff
@@ -3,12 +3,20 @@
     <b-row>
       <b-col cols="12" lg="6">
         <label>Usersuche</label>
-        <b-input
-          type="text"
-          v-model="criteria"
-          class="shadow p-3 mb-5 bg-white rounded"
-          placeholder="User suche"
-        ></b-input>
+        <b-input-group>
+          <b-form-input
+            type="text"
+            class="test-input-criteria"
+            v-model="criteria"
+            :placeholder="$t('user_search')"
+          ></b-form-input>
+
+          <b-input-group-append class="test-click-clear-criteria" @click="criteria = ''">
+            <b-input-group-text>
+              <b-icon icon="x" />
```

Comment:
The mouse cursor should be a pointer.

Reply:
https://user-images.githubusercontent.com/1324583/152492480-21f07290-6a85-498b-9f09-e80e470494f6.mp4